qiskit_ibm_runtime/options/resilience_options.py

```diff
@@ -100,8 +100,15 @@ def _validate_options(self) -> "ResilienceOptionsV2":
         # Validate not ZNE+PEC
         if self.pec_mitigation is True and self.zne_mitigation is True:
             raise ValueError(
-                "pec_mitigation and zne_mitigation`options cannot be "
+                "'pec_mitigation' and 'zne_mitigation' options cannot be "
                 "simultaneously enabled. Set one of them to False."
             )
 
+        if not isinstance(self.layer_noise_model, UnsetType) and None in [
```

Comment:
I would change this to allow values of None in the options for two reasons:
1. It lets users pass in the returned option from one estimator/noise_learner run to another Estimator run call without having to manually modify the options and remove any None values 
2. Maybe some uers will want to experiment with manually disabling mitigation for specific layer(s) for debugging/research applications?

Reply:
I removed this if because it's not needed. Server doesn't crush